modules/distribution/src/assembly/bin.xml

```diff
@@ -685,38 +589,21 @@
              <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/tomcat</outputDirectory>
              <filtered>true</filtered>
              <fileMode>644</fileMode>
-         </file>
+         </file>-->
         <file>
             <source>src/repository/conf/tomcat/context.xml</source>
             <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/tomcat</outputDirectory>
             <filtered>true</filtered>
             <fileMode>644</fileMode>
         </file>
-        <!--<file>-->
-        <!--<source>-->
-        <!--../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/device-type-config.xsd-->
-        <!--</source>-->
-        <!--<outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/iot</outputDirectory>-->
-        <!--<filtered>true</filtered>-->
-        <!--<fileMode>644</fileMode>-->
-        <!--</file>-->
-        <!--<file>-->
-        <!--<source>-->
-        <!--../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/devicecloud-config.xml-->
-        <!--</source>-->
-        <!--<outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/iot</outputDirectory>-->
-        <!--<filtered>true</filtered>-->
-        <!--<fileMode>644</fileMode>-->
-        <!--</file>-->
+        <!--        <file>
+                    <source>../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/event-broker.xml
+                    </source>
+                    <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/</outputDirectory>
+                    <filtered>true</filtered>
+                    <fileMode>644</fileMode>
+                </file>-->
```

Comment:
Already fixed in commit https://github.com/wso2-incubator/product-iot-server/commit/febdc36cba8b94c3ebcaba27947893ebb2865513 as per the PrabathA's comment.